src/routes/login/index.tsx

```diff
@@ -50,7 +55,12 @@ class Login extends Component<{}, LoginState> {
               onInput={linkState(this, 'password')}
               value={state.password}
             />
-            <Button type="submit" value="Login" />
+            <div class={buttons}>
+              <Button>Login</Button>
```

Comment:
Can you put the login button on the right and make the register button a lot lighter grey and remove the shadow? That way login is more clearly the "default" action and register is the less recommended action.